4_processing_features/Plate2/CellProfiler/extract_sc_features_cp_plate2.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
```

Comment:
Oh geez you make a good point here, I think this would need a much more in depth discussion since I can not think of a way to automate this currently.

Reply:
This is now an issue in the repo @d33bs